enhancements/declarative-index-config.md

```diff
@@ -59,7 +59,8 @@ A human consumable representation of packages in an index in the form of json/ya
 ## Non-goals 
 
 1. Enumerate/implement tooling to allow different operations to be performed on the package representations. Numerous tooling(eg those that allow operation on files) that already exists can be leveraged to perform various operations on package representations. 
-2. Advanced tooling to verify dependency satisfiability for individual bundles, channel upgrade graph validity for each channel in a package etc. This will be considered in a separate enhancement.      
+2. Advanced tooling to verify dependency satisfiability for individual bundles, channel upgrade graph validity for each channel in a package etc. This will be considered in a separate enhancement.
+3. Define an exhaustive or authoritative set of bundle properties. This proposal will document only the properties necessary to maintain backwards-compatibility with the existing GRPC API.
 
```

Comment:
We are saying here that OLM will be looking always for the json (index) and then, that we can edit that by using opm. However, the real bundle will still with the original values in https://github.com/operator-framework/community-operators. So, what problems we might face with that?

- If only the admin catalogue maintainer can do these changes, how the operator author will be able to change the values?
- How an operator author can check if the project ha been distributed as it was configured?
- Then, it means that we will not be able to check/troubleshooting by looking at the data in https://github.com/operator-framework/community-operators as well. Am I right? 
- Should we not provide a way for the unit/operator bundle to be changed in te process by the author instead of the index catalogue admin? Are we not here giving more manual work for the pipeline team/catalogue admins instead of providing solutions that allow the process to be fully automated and without requiring manual changes in the index catalogue image?